PostFreeSurfer/scripts/convert_tp_to_template_xfm

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then 
```

Comment:
Should we convert to Tim's library options?

Reply:
excluded this script.